spark-3.1-spanner-lib/src/main/resources/spark-spanner-connector.properties

```diff
@@ -0,0 +1,2 @@
+#connector.version=${project.version}
+connector.version=3.1
```

Comment:
Any reason for that? the idea is to track the connector's version for the user agent